lib/index.js

```diff
@@ -56,12 +57,16 @@ module.exports = function(playlistURL, options) {
   }
 
   var tid;
+  var rpts;
```

Comment:
rpts means "Repeat Times". It's the amount of times refreshPlaylist() has been called when ytdl-error runs. If it loops more than 5 times from ytdl-errors in a row, then it's probably crashing and time to stop.